improver/developer_tools/metadata_interpreter.py

```diff
@@ -337,9 +337,15 @@ def _check_blend_and_model_attributes(self, attrs: Dict) -> None:
                     f"expected substring {BLEND_TITLE_SUBSTR}."
                 )
 
-    def check_attributes(self, attrs: Dict) -> None:
+    @staticmethod
+    def _cubeattrsdict_as_dict(attrs: CubeAttrsDict) -> dict:
+        """Returns a dict from a CubeAttrsDict, because it has preferable str() methods"""
+        return {key: value for key, value in attrs.items()}
```

Comment:
You can just call `dict(cube.attributes)` to get the standard dict form, so perhaps this is superfluous?